eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -21,15 +21,14 @@ parameters:
     default: false
 
 steps:
-  # Internal pipeline: Build with pack+sign+archive
+  # Internal pipeline: Build with pack+sign
   - ${{ if ne(parameters.runAsPublic, 'true') }}:
     - script: ${{ parameters.buildScript }}
               -restore -build
               -pack
               -sign $(_SignArgs)
               -publish $(_PublishArgs)
               -configuration ${{ parameters.buildConfig }}
-              /p:ArchiveTests=true
```

Comment:
I'm not following, why are we no longer zipping the tests?

Reply:
Why do we need to zip tests if we aren't giong to run them?

Reply:
Yep, the tests are zipped only for sending to helix.

Reply:
I thought this template was used by both our public CI and official builds, but maybe that's not the case then?

Reply:
Ohhhhh just saw the condition above for not run as public... duh! don't mind me, I can't read apparently 😝

Reply:
There is a conditional step below this for public builds.